vpc.tf

```diff
@@ -12,4 +12,13 @@ resource "aws_subnet" "core-infra" {
 
 resource "aws_internet_gateway" "gw"{
   vpc_id = aws_vpc.core-infra.id
+}
+
+resource "aws_nat_gateway" "gw"{
```

Comment:
https://www.terraform.io/docs/providers/aws/r/internet_gateway.html

not nat gateway

Reply:
you told me to create nat gateway and attach eip to it, so do i need the ```resource "aws_nat_gateway" "gw"{
  allocation_id = aws_instance.bastion-host.id
  subnet_id = aws_subnet.core-infra.id

  tags = {
    Name = "nat gateway"
  }
}```